pandas/core/indexing.py

```diff
@@ -1788,7 +1787,6 @@ def isetter(loc, v):
             self.obj._check_is_chained_assignment_possible()
 
             # actually do the set
-            self.obj._consolidate_inplace()
```

Comment:
Removing this one will need a performance check, IMO, and also has potential API changes that might need investigation